security/monitoring.adoc

```diff
@@ -0,0 +1,202 @@
+resource they are related to or, for cluster events, the *_default_*
+namespace. The *_default_* namespace is likely to hold the most relevant
+events for monitoring or auditing a cluster, such as *_Node_* events and resource events related to
+infrastructure components.
+
+The master API and `oc` command do not provide parameters to scope a listing of events to only those
+related to nodes. A simple approach would be to use grep:
+
+[source, bash]
+----
+$ oc get event -n default | grep Node
+1h         20h         3         origin-node-1.example.local   Node      Normal    NodeHasDiskPressure   ...
+----
+
+A more flexible approach is to output the events in a form that other
+tools can process. For example, the following example uses the `*jq*`
+tool against JSON output to extract only *_NodeHasDiskPressure_* events:
+
+[source, bash]
+----
+$ oc get events -n default -o json \
+  | jq '.items[] | select(.involvedObject.kind == "Node" and .reason == "NodeHasDiskPressure")'
+
+{
+  "apiVersion": "v1",
+  "count": 3,
+  "involvedObject": {
+    "kind": "Node",
+    "name": "origin-node-1.example.local",
+    "uid": "origin-node-1.example.local"
+  },
+  "kind": "Event",
+  "reason": "NodeHasDiskPressure",
+  ...
+}
+----
+
+Events related to resource creation, modification, or deletion globally
+can also be good candidates for detecting misuse of the cluster. As an
+example, to look for excessive pulling of images, the following query
+counts the number of such recent events:
+
+[source, bash]
+----
+$ oc get events --all-namespaces -o json \
+  | jq '[.items[] | select(.involvedObject.kind == "Pod" and .reason == "Pulling")] | length'
+
+4
+----
+
+[[NOTE]]
+====
+When a namespace is deleted, its events are deleted along with it. Events also expire and are removed to prevent filling up *etcd*. Events are not stored
+as a permanent record and frequent polling is necessary to capture statistics over time.
+====
+
+[[security-monitoring-logs]]
+== Cluster Logs
+
+[[security-monitoring-service-logs]]
+=== Service Logs
+
+{product-title} produces logs with each *systemd* service that is running on a host:
+
+ifdef::openshift-origin[]
+- origin-master-api
+- origin-master-controllers
+- etcd
+- origin-node
+endif::[]
+ifdef::openshift-enterprise[]
+- atomic-openshift-master-api
+- atomic-openshift-master-controllers
+- etcd
+- atomic-openshift-node
+endif::[]
+
+These logs provide a relatively verbose record of component operations
+directed more at debugging purposes than security auditing. They can be
+retrieved per host with `*journalctl*` or, in clusters with the aggregated
+logging stack deployed, in the logging _.operations_ indexes (possibly
+in the xref:../install_config/aggregate_logging.adoc#aggregated-ops[Ops
+cluster]) as a cluster administrator.
+
+[[security-monitoring-audit-log]]
+=== Master API Audit Log
+
+For security auditing purposes, there is an option to enable
```

Comment:
Again, this is way more wordy that we need to be.  Consider rewording - an example:

To log master API requests by users, administrators, or system components enable audit logging  +xref:../install_config/master_node_configuration.adoc#master-node-config-audit-config[for the master API]. This will create a file on each master host or, if there is no file configured, be included in the service's journal. Entries in the journal can be found by searching for "AUDIT".